posthog/settings.py

```diff
@@ -59,6 +59,7 @@ def print_warning(warning_lines: Sequence[str]):
 
 DEBUG = get_bool_from_env("DEBUG", False)
 TEST = "test" in sys.argv  # type: bool
+SELF_CAPTURE = get_bool_from_env("SELF_CAPTURE", False)
```

Comment:
Maybe if no env value is provided default to `True if DEBUG else False`? That way on posthog/utils.py#190 you only use `settings.SELF_CAPTURE` which would be a bit cleaner.